toolchain/sem_ir/name_scope.h

```diff
@@ -67,33 +67,53 @@ class ScopeLookupResult {
   // True when lookup was successful or resulted with an error. False for
   // poisoned or not found.
   auto is_found() const -> bool {
-    return !is_poisoned() && inst_id_.has_value();
+    return !is_poisoned() && target_inst_id_.has_value();
   }
 
   // The `InstId` of the result of the lookup. Must only be called when lookup
   // was successful e.g. `is_found()` returns true. Always returns an existing
   // `InstId`.
   auto target_inst_id() const -> InstId {
     CARBON_CHECK(is_found());
-    return inst_id_;
+    return target_inst_id_;
+  }
+
+  // The `LocId` where the name poisoning was triggered. Must only be called
+  // when lookup resulted with a poisoned name e.g. `is_poisoned()` returns
```

Comment:
```suggestion
  // when lookup returned a poisoned name; in other words, when `is_poisoned()` returns
```

"e.g." means "for example"
"i.e." means "that is" or "in other words"

I think "for example" is incorrect here; "is_poisoned" is instead restating the same thing in a different way. It may be helpful to avoid the abbreviation. (plus other minor grammar)

Reply:
Done.